assets/js/base/context/cart-checkout/checkout/processor/index.js

```diff
@@ -182,23 +185,33 @@ const CheckoutProcessor = () => {
 							);
 						}
 						dispatchActions.setHasError();
+					} else {
+						dispatchActions.setRedirectUrl(
+							response.payment_result.redirect_url
+						);
 					}
 
-					dispatchActions.setRedirectUrl(
-						response.payment_result.redirect_url
-					);
 					dispatchActions.setComplete();
+					setIsProcessingOrder( false );
 				} );
 			} )
 			.catch( ( error ) => {
-				addErrorNotice( error.message, {
+				const message =
+					error.message ||
+					__(
+						'Something went wrong. Please contact the site administrator to get assistance.',
```

Comment:
Here, I rewrote the text to suggest contacting the site administrator instead of "try again" (as suggested in an error some lines above :point_up: ). If I'm understanding it correctly, the payment has already been done but the purchase could not be completed, so we shouldn't suggest users to purchase it again, otherwise they would pay twice. Does that make sense?

Reply:
Since this is customer facing we should say "Please contact us to get assistance" rather than "Please contact the site administrator to get assistance" which is pretty unfriendly.

Reply:
Good point. I will change it. :+1: 

What about this one:
https://github.com/woocommerce/woocommerce-gutenberg-products-block/blob/master/assets/js/base/context/cart-checkout/checkout/processor/index.js#L176
would make it sense changing it too?

Reply:
Depends if it's fixable by the user :p

Reply:
Even if the user can fix it, when `processOrder` runs, the payment has already been done, no? In that case I think we shouldn't suggest the user to try again to avoid the payment being processed again.

Reply:
I changed the text of the errors in: 49980c291c8ff30f2537dc9a3c138692cb54d4b5.